pkg/net/socketpair_windows.go

```diff
@@ -55,15 +47,14 @@ func emulateWithPreConnect() (SocketPair, error) {
 		err error
 	)
 
+	syscall.ForkLock.RLock()
+	defer syscall.ForkLock.RUnlock()
```

Comment:
I think the lock is too big here. would you please use a wrap to handle each syscall?

REF: https://github.com/golang/go/blob/d7df7f4fa01f1b445d835fc908c54448a63c68fb/src/net/sock_windows.go#L31

Reply:
Will be done in windows PR

Reply:
PR for windows: https://github.com/containerd/nri/pull/69